packages/object-map/index.d.ts

```diff
@@ -1,6 +1,6 @@
-declare function map<T, U = T>(
-  item: Record<string, T>,
-  callback: (key: string, value: T) => U
-): Record<string, U>;
+declare function map<K extends string | number | symbol, T, U = T>(
```

Comment:
I don't think object keys can ever be numbers

Reply:
hey! sorry for the late reply. you're right, that's my mistake. i updated the change to remove `number` from the type